guides/common/modules/proc_exporting-a-repository-incrementally-in-a-syncable-format.adoc

```diff
@@ -0,0 +1,31 @@
+[id="Exporting_Repository_Incrementally_in_a_Syncable_Format_{context}"]
+= Exporting a Repository Incrementally in a Syncable Format
+
+Exporting a repository can be a very costly operation in terms of system resources.
+A typical {RHEL} tree may occupy several gigabytes of space on {ProjectServer}.
+
+In such cases, you can use *Incremental Export* to export only pieces of content that changed since the previous export.
+Incremental exports typically result in smaller archive files than full exports.
+
+The procedure below shows an incremental export of a repository in the Library lifecycle environment.
+
+.Procedure
+. Create an incremental export:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer content-export incremental repository \
+ --organization="_My_Organization_" \
+ --product="_My_Product_" \
+ --name="_My_Repository_" \
+ --format=syncable
+Generated /var/lib/pulp/exports/Default_Organization/Export-SYNCABLE-Red_Hat_Ansible_Engine_2_for_RHEL_8_x86_64_RPMs-1/2.0/2023-03-09T10-55-48-05-00/metadata.json
+----
+. Optional: View the exported data:
++
+[options="nowrap" subs="+quotes"]
+----
+# cd /var/lib/pulp/exports/Default_Organization/Export-SYNCABLE-Red_Hat_Ansible_Engine_2_for_RHEL_8_x86_64_RPMs-1/2.0/2023-03-09T10-55-48-05-00/
+# find . -name *.rpm
+./content/dist/layered/rhel8/x86_64/ansible/2/os/Packages/s/sshpass-1.06-3.el8ae.x86_64.rpm
```

Comment:
```suggestion
# find /var/lib/pulp/exports/Default_Organization/Export-SYNCABLE-Red_Hat_Ansible_Engine_2_for_RHEL_8_x86_64_RPMs-1/2.0/2023-03-09T10-55-48-05-00/ -name "*.rpm"
```

If you're worried about showing a very long path for each package, you could add `| xargs basename -a` which strips with path and only shows the RPM file name.